diff --git a/ld/configure.tgt b/ld/configure.tgt
index 23194e357f..5953f0a119 100644
--- a/ld/configure.tgt
+++ b/ld/configure.tgt
@@ -387,7 +387,7 @@ i[3-7]86-*-pe)		targ_emul=i386pe ;
 			;;
 i[3-7]86-*-cygwin*)	targ_emul=i386pe ;
 			targ_extra_ofiles="deffilep.o pe-dll.o" ;
-			test "$targ" != "$host" && LIB_PATH='${tooldir}/lib/w32api'
+			test "$targ" != "$host" && LIB_PATH='${tooldir}/lib/w32api:=/usr/lib/w32api'
 			;;
 i[3-7]86-*-mingw32*)	targ_emul=i386pe ;
 			targ_extra_ofiles="deffilep.o pe-dll.o"
@@ -991,7 +991,7 @@ x86_64-*-pe | x86_64-*-pep) targ_emul=i386pep ;
 x86_64-*-cygwin)	targ_emul=i386pep ;
 			targ_extra_emuls=i386pe
 			targ_extra_ofiles="deffilep.o pep-dll.o pe-dll.o"
-			test "$targ" != "$host" && LIB_PATH='${tooldir}/lib/w32api'
+			test "$targ" != "$host" && LIB_PATH='${tooldir}/lib/w32api:=/usr/lib/w32api'
 			;;
 x86_64-*-mingw*)	targ_emul=i386pep ;
 			targ_extra_emuls=i386pe
